core-data/queried-data/constant.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Action type used in signalling that a queried items page has been received.
+ *
+ * @type {string}
+ */
+export const RECEIVE_ITEMS = '@@queried-data/RECEIVE_ITEMS';
+
+/**
+ * Action type used in signalling that a request has started or stopped for an
+ * items query.
+ *
+ * @type {string}
+ */
+export const TOGGLE_IS_REQUESTING = '@@queried-data/TOGGLE_IS_REQUESTING';
```

Comment:
>Any specific reason to introduce the constants. Not that I'm strongly against them but I do wonder about having some actions defined as constant and others not.

It was implemented in the mindset of being an independent module utility, with prefixes on the action types to avoid potential conflicts. In earlier iterations, I had the consumer listening for the particular action types, where I didn't want them to be concerned with said prefixes. Since now only the internal reducer is concerned, we could just replace them with strings. Ultimately, I don't consider these files to be part of `core-data` or the `data` module patterns whatsoever, so the conventions don't overlap as much as it might appear on the surface.

Maybe a bit premature to modularize this, but having reimplemented effectively the same thing on [many](https://github.com/Automattic/wp-calypso/tree/master/client/lib/query-manager) [separate](https://github.com/Automattic/wp-calypso/blob/140941cccaf7a87699dead4fbf6830271c76c5c1/client/state/posts/reducer.js#L130-L151) [occasions](https://github.com/aduth/dones/blob/0450c88429deb4546f72d20f915551ec62a33de3/src/state/dones/reducer.js#L76-L111), I was motivated toward something more universal 😅